docs/fides/docs/development/postman/Fides.postman_collection.json

```diff
@@ -1,8 +1,8 @@
 {
 	"info": {
-		"_postman_id": "71013d58-9646-47e3-b47a-f354d33ad35a",
+		"_postman_id": "5d1a5ee9-f432-4da2-a9c9-d68c17e776ac",
 		"name": "Fides",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
```

Comment:
looks like theres an additional `,` here that's causing parsing errors when i try to import into postman. just removing the trailing `,` made things work.